samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SaveAssessmentSettings.java

```diff
@@ -353,7 +361,14 @@ public AssessmentFacade save(AssessmentSettingsBean assessmentSettings, boolean
     assessment.setTitle( newTitle );
     assessment.updateAssessmentMetaData(SecureDeliveryServiceAPI.TITLE_DECORATION, titleDecoration );
 
-    assessment.setInstructorNotification(Integer.valueOf(assessmentSettings.getInstructorNotification()));
+    try
+    {
+        assessment.setInstructorNotification(Integer.valueOf(assessmentSettings.getInstructorNotification()));
+    }
+    catch( NullPointerException | NumberFormatException ex )
+    {
+        LOG.warn( ex );
```

Comment:
In the case of a NPE or NFE, should the assessment Instructor Notification be set to the [default](https://github.com/sakaiproject/sakai/blob/4d16ce1a21aa50e4335d5a64d00660a6f86c7f0b/samigo/samigo-api/src/java/org/sakaiproject/samigo/util/SamigoConstants.java#L57)?